components/application-tag-mgt/org.wso2.carbon.identity.application.tag.common/src/main/java/org/wso2/carbon/identity/application/tag/common/model/ApplicationTagPOST.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.application.tag.common.model;
+
+/**
+ * Application Tag POST.
```

Comment:
This does not explain the purpose of the class.

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/4904/commits/de7f9a97c7f701696ad692435fc471125e647f71